api/proto/teleport/legacy/types/types.proto

```diff
@@ -6175,3 +6179,9 @@ message KubernetesMatcher {
     (gogoproto.customtype) = "Labels"
   ];
 }
+
+// OktaOptions specify options related to the Okta service.
+message OktaOptions {
+  // TimeBetweenSyncs is the duration between synchronization calls.
+  int64 TimeBetweenSyncs = 1 [(gogoproto.jsontag) = "seconds_between_syncs,omitempty"];
```

Comment:
- odd to use "seconds between syncs" in JSON but "time between syncs" for the field name, no?
- how about `SyncPeriod` (`sync_period`) instead?

Reply:
Argh, I meant for this to be `TimeBetweenSyncs` for both. I thought I had excised it all. Whoops. Will change.